src/renwindow.h

```diff
@@ -6,19 +6,17 @@ struct RenWindow {
 #ifdef LITE_USE_SDL_RENDERER
   SDL_Renderer *renderer;
   SDL_Texture *texture;
-  SDL_Surface *surface;
-  int surface_scale;
```

Comment:
`RenSurface` appears to only be a combination of `surface` and `surface_scale` with a changed datatype for scales.

Does this make working with surfaces independently easier?
Does it only affect the use of the SDL Renderer?

Reply:
Yes, it dissociates the surface from the window scale.
At the moment only using the SDL Renderer we get a scale that might be not 1, yeah.

I think in the future, when we refactor how we scale, we'll just not need to have a scale variable here, so then we might just replace `RenSurface` with `SDL_Surface`.

Reply:
Ah, alright.

Is switching to a fixed point number going to introduce any problems?

Reply:
Hopefully not.
This change was done thinking about future fractional scaling support. But considering that we might just scrap the whole surface scale concept, I should probably just revert this to be an `int` like before.